content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -323,6 +323,53 @@ the files in the following order:
 1. `support/component.js` (your support file)
 2. `components/Button/Button.cy.js` (your spec file)
 
+#### Use supportFile to load scripts before your test code
```

Comment:
This may seem like a silly question, but do you know why we're adding this section?

Reply:
Specifically the list of files that the supportFile looks for...or just this whole section?

Reply:
The whole section. I mean, I'm reading this now, and because the `supportFile` only has a different default filename but isn't actually doing anything different in 10.0, why did we decided to add this new content?

Reply:
This is more expanding the list of files and moving this content vs. adding anything new. It's currently in production on [the error messages page](https://docs.cypress.io/guides/references/error-messages#Use-supportFile-to-load-scripts-before-your-test-code). One of your comments in this PR was to move it here since it's weird beside errors. 

Then we were playing around with how to format it and there were some other tweaks around this stuff in other PRs so it might feel out of place a bit as-is now.